templates/compass.html

```diff
@@ -31,6 +31,10 @@ <h2 style="margin-top:-55px">find your way</h2>
                             </div>
                             <button id="generateLayer" type="button" class="button-success"
                                     style="" onclick="generateLayer()">Generate Layer</button>
+                            <input id='adversaryLayerInput' type='file' hidden/>
+                            <button id="generateAdversary" type="button" class="button-success"
```

Comment:
these buttons are too far apart. but overall, now that this modal section can do two things, you should consider redesigning the way it looks. no need to copy the 2-column look of most of the modals. this one may need something a bit more custom to look right.

Reply:
updated
![Untitled](https://user-images.githubusercontent.com/18699738/72000311-05b80400-3211-11ea-96d3-adf4f9025a0e.png)